src/components/dock/dock-button/dock-button.tsx

```diff
@@ -124,14 +124,22 @@ export class DockButton {
                     selected: this.item?.selected,
                 }}
                 onClick={handleClick}
+                aria-live={this.item.badge ? 'polite' : 'off'}
             >
                 {this.renderIcon()}
                 {this.renderLabel()}
                 {this.renderTooltip()}
+                {this.renderNotification()}
             </button>
         );
     }
 
+    private renderNotification = () => {
+        if (this.item.badge) {
```

Comment:
yes if you ask me. But that depends on how we do in limel-badge, or?

Reply:
limel-badge actually allows label to be empty or missing already and it will be rendered the same way. It was only the docs that suggested that label was required I think.

```suggestion
        if (this.item.badge || this.item.badge === '') {
```

Reply:
Ah OK. Cool. Yes 👇 
<img width="932" alt="image" src="https://user-images.githubusercontent.com/35954987/220361621-8be57bf7-145b-4581-8248-c0ef8a7e3aea.png">
I just think it unnecessarily renders too big. I'll change this in a new PR.